iterative_verifier.go

```diff
@@ -133,6 +133,10 @@ type IterativeVerifier struct {
 	SourceDB         *sql.DB
 	TargetDB         *sql.DB
 
+	// This makes the verifier ignore any problems and assume all rows match,
+	// so that performance can be tested without running a full copy.
```

Comment:
how can we test performance if we don't copy any data? doesn't that skew the numbers a lot (because the hashes in the target will all be empty)?

Reply:
The biggest limiting factor is latency at the moment, our pipeline utilization is pretty low. The values from the dryrun are showing ~41.8Mrows/hour while the last failed move was ~37.2Mrows/hour. So ~10% faster. The data should still be useful.